released/packages/coq-fourcolor-reals/coq-fourcolor-reals.1.4.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+
+homepage: "https://github.com/coq-community/fourcolor"
+dev-repo: "git+https://github.com/coq-community/fourcolor.git"
+bug-reports: "https://github.com/coq-community/fourcolor/issues"
+license: "CECILL-B"
+
+synopsis: "Interface for real numbers used in the Four Color Theorem"
+description: """
+An axiomatization of the setoid of classical real numbers, along
+with proofs of properties such as categoricity."""
+
+build: [make "-C" "theories/reals" "-j%{jobs}%"]
+install: [make "-C" "theories/reals" "install"]
+depends: [
+  "coq" {(>= "8.16" & < "8.21~") | (= "dev")}
+  "coq-mathcomp-ssreflect" {(>= "2.0.0") & < "2.4~") | (= "dev")}
+  "coq-mathcomp-algebra" 
+]
+
+tags: [
+  "category:Mathematics/Real Calculus and Topology"
+  "keyword:real numbers"
+  "logpath:fourcolor.reals"
+]
+
+"date: 2024-11-15"
```

Comment:
We should prevent installation of this package together with earlier fourcolor (they use the same directory).
```suggestion
  "logpath:fourcolor.reals"
  "date: 2024-11-15"
]
conflicts: [
  "coq-fourcolor" { != version }
]
```